judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/submission/bundle/Verdict.java

```diff
@@ -0,0 +1,15 @@
+package judgels.sandalphon.api.submission.bundle;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableVerdict.class)
+public interface Verdict {
+    String getCode();
+    String getName();
```

Comment:
This is legacy way from programming. Actually we don't have to store the name in DB as it is actually a derived value. Frontend should provide the name is required. So please remove the name.

Reply:
OK - changed Verdict to enum